### ISSUE TITLE
TypeError: LassoSelector

### ISSUE THREAD

**Issue by benajamin** [mlerotic/spectromicroscopy#12]
When clicking on the "Select ROI (Lasso)" button, I'm not able to draw a lasso onto the image and the following error is thrown:
```
Traceback (most recent call last):
  File "/home/watts/dev/mantis/spectromicroscopy/mantis_xray/mantis_qt.py", line 10046, in OnAddROI
    self.lasso = LassoSelector(self.axes, onselect=self.OnSelectLasso, useblit=False, lineprops=lineprops)
  File "/home/watts/.local/lib/python3.10/site-packages/matplotlib/_api/deprecation.py", line 454, in wrapper
    return func(*args, **kwargs)
TypeError: LassoSelector.__init__() got an unexpected keyword argument 'lineprops'
```

**Comment by benajamin** [comment]
The 'lineprops' arguement was changed to props in Matplotlib version 3.5.

**Comment by benajamin** [comment]
fixed in commit [92ab097](https://github.com/mlerotic/spectromicroscopy/commit/92ab097775f71b314cc3ff058952e14d6ee146e8) for version 3.1.13
